Documentation ko

kyeongwan opened the pull request:
The PR should summarize what was changed and why. Here are some questions to
help you if you're not sure:

 - What behavior was changed?
 - What code was refactored / updated to support this change?
 - What issues are related to this PR? Or why was this change introduced?

Checklist - While not every PR needs it, new features should consider this list:

 - [ ] Does this have tests?
 - [ ] Does this have documentation?
 - [ ] Does this break the public API (Requires major version bump)?
 - [ ] Is this a new feature (Requires minor version bump)?

kkw567 committed:
Added documentation Korean translation

kkw567 committed:
Make :kr: translations more smoother

yangroro commented on Documentation/Ko-kr/ArrangeActAssert.md:
```diff
@@ -0,0 +1,203 @@
+우리의 프로그램 코드는 다음 두 가지 중 하나를 수행합니다 :
+
+1. 이미 벗겨진(먹을 수 있는) 바나나를 제공합니다…
+2. …또는 벗겨지지 않은(먹을 수 없는) 바나나를 제공합니다.
```
이 부분에서 또는 이 필요한지 의문입니다. ...이라는 문장 부호도 마찬가지고요.

kyeongwan replied:
문장 부호는 그대로 둘 것입니다.
타 언어로 번역된 (일어, 중어) 도 문장기호를 그대로 유지했더라구요

yangroro commented on Documentation/Ko-kr/InstallingQuick.md:
```diff
@@ -0,0 +1,141 @@
+
+https://github.com/Quick/QuickOnLinuxExample
+
+### (권장되지 않는 방법) 실제 iOS 기기에서 Quick Specs를 실행하기
```
`(권장하지 않는 방법)` 이 더 자연스럽지 않을까요

kyeongwan replied:
수정하겠습니다.
약간 수동태 같은 문장이네요.
이왕 이렇게 된거 맞춤법 검사기 한번 돌리겠습니다.

yangroro commented on Documentation/Ko-kr/QuickExamplesAndGroups.md:
```diff
@@ -0,0 +1,462 @@
+
+class DolphinSpec: QuickSpec {
+  override func spec() {
+    it("친절한가") {
```
이부분만 왜 한글인가요? 일관성을 해치는 것 같습니다.

kkw567 committed:
Fix korean document typo.

- Modified according to Korean grammar
- Change all code to English (for consistency)

kkw567 committed:
Add Ko-kr (한국어)
